src/controllers/commands/CommandController.cpp

```diff
@@ -3211,6 +3211,111 @@ void CommandController::initialize(Settings &, Paths &paths)
+
+        using Error = HelixSendShoutOutError;
+
+        getHelix()->getUserByName(
+            target,
+            [twitchChannel, channel, currentUser,
+             &target](const auto targetUser) {
+                getHelix()->sendShoutout(
+                    twitchChannel->roomId(), targetUser.id,
+                    currentUser->getUserId(),
+                    [channel, targetUser]() {
+                        channel->addMessage(
+                            makeSystemMessage(QString("Sent shoutout to %1")
+                                                  .arg(targetUser.login)));
+                    },
+                    [channel](auto error, auto message) {
+                        QString errorMessage = "Failed to send shoutout - ";
+                        qCInfo(chatterinoTwitch) << message;
+
+                        switch (error)
+                        {
+                            case Error::UserNotAuthorized: {
+                                errorMessage += "You don't have permission to "
+                                                "perform that action.";
+                            }
+                            break;
+
+                            case Error::UserMissingScope: {
+                                errorMessage += "Missing required scope. "
+                                                "Re-login with your "
+                                                "account and try again.";
+                            }
+                            break;
+
+                            case Error::Ratelimited: {
+                                errorMessage +=
+                                    "You are being ratelimited by Twitch. "
+                                    "Try again in a few seconds.";
+                            }
+                            break;
+
+                            case Error::UserNotMod: {
+                                errorMessage +=
+                                    "You are not a moderator of this channel.";
+                            }
+                            break;
+
+                            case Error::UserIsBroadcaster: {
+                                errorMessage += "The broadcaster may not give "
+                                                "themselves a Shoutout.";
+                            }
+                            break;
+
+                            case Error::BroadcasterNotLive: {
+                                errorMessage +=
+                                    "The broadcaster is not streaming live or "
+                                    "does not have one or more viewers.";
+                            }
+                            break;
+
+                            case Error::Unknown: {
+                                errorMessage +=
+                                    "An unknown error has occurred.";
+                            }
+                            break;
```

Comment:
I would be ok with including the `message` portion in unhandled errors here, so the user would see something like "Internal server error" or w/e if we don't handle one of the error cases.